src/compat/compat.h

```diff
@@ -21,7 +21,7 @@
 
 #include "export.h"
 
-#define PG_MAJOR_MIN 12
+#define PG_MAJOR_MIN 13
 
 #define is_supported_pg_version_12(version) ((version >= 120000) && (version < 130000))
```

Comment:
is this still needed?

Reply:
Yes, for now. PG12 is defined as is_supported_pg_version_12 and I plan to remove that PG12 macro in a separate commit.

Reply:
There is also another followup to this patch that removes the testfiles and update files - https://github.com/timescale/timescaledb/pull/5865. I plan to handle the PG12 Macros next.